pkg/cmd/server/start/start_api.go

```diff
@@ -70,7 +70,6 @@ func NewCommandStartMasterAPI(name, basename string, out io.Writer) (*cobra.Comm
 	// This command only supports reading from config and the listen argument
 	flags.StringVar(&options.ConfigFile, "config", "", "Location of the master configuration file to run from. Required")
 	cmd.MarkFlagFilename("config", "yaml", "yml")
-	// TODO: add health and metrics endpoints on this listen argument
 	BindListenArg(options.MasterArgs.ListenArg, flags, "")
```

Comment:
I did remove that one... I thought...

On Mon, Aug 17, 2015 at 2:55 PM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/server/start/start_api.go
> <https://github.com/openshift/origin/pull/4137#discussion_r37223297>:
>
> > @@ -70,7 +70,6 @@ func NewCommandStartMasterAPI(name, basename string, out io.Writer) (*cobra.Comm
> >  	// This command only supports reading from config and the listen argument
> >  	flags.StringVar(&options.ConfigFile, "config", "", "Location of the master configuration file to run from. Required")
> >  	cmd.MarkFlagFilename("config", "yaml", "yml")
> > -	// TODO: add health and metrics endpoints on this listen argument
> >  	BindListenArg(options.MasterArgs.ListenArg, flags, "")
>
> Didn't you agree to remove this one and leave the one on start_controllers?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4137/files#r37223297>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift